db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"self.first_name self.last_name"
```

Comment:
The `__str__` method is returning a string with the literal text `self.first_name self.last_name`. To correctly return the actor's full name, it should be formatted as `f'{self.first_name} {self.last_name}'`.